legate/core/_legion/operation.py

```diff
@@ -1145,14 +1145,16 @@ def __init__(
         self._launcher = ffi.gc(
             self.launcher, legion.legion_attach_launcher_destroy
         )
+        if not data.c_contiguous and not data.f_contiguous:
+            raise RuntimeError("Can only attach to C- or F-contiguous buffers")
         legion.legion_attach_launcher_add_cpu_soa_field(
             self.launcher,
             ffi.cast(
                 "legion_field_id_t",
                 field.fid if isinstance(field, FieldID) else field,
             ),
             ffi.from_buffer(data),
-            data.f_contiguous,
+            not data.c_contiguous,
```

Comment:
What case is missed by the original code, under the condition that at this point, `data` is asserted to be `contiguous` ?

Reply:
0d and 1d contiguous array are considered to be both `c_contiguous` and `f_contiguous`, so in that case I wanted to tell the attachment operation to take the C-contiguous path. Not that it matters very much.

Reply:
Ah that makes sense, a short comment might be advised (and maybe also to the effect that it doesn't really matter)

Reply:
Yup, added suggested comments with f01cce9485a47c8287b59975e3ea39b792be4bcc